src/actions/slack/utils.ts

```diff
@@ -219,8 +233,45 @@ export const handleExecute = async (request: Hub.ActionRequest, slack: WebClient
             })
         }
     } catch (e: any) {
-        winston.error(`${LOG_PREFIX} Slack App Error: ${e.message}`)
-        response = new Hub.ActionResponse({success: false, message: e.message})
+        let error: Error = errorWith(
```

Comment:
This assignment here should be the else on the if/else clause below to avoid creating two error objects here.

Reply:
How would it create two error objects? Wouldn't it just override the variable error if any of the conditions were met?

Reply:
We create an error object when we call errorWith - in this code as it is set up, we are creating a baseline object with HTTP_ERROR.internal.  Depending on the code afterwards, we may potentially replace it with another object.  This is a small update, but I think makes the code easier to represent if we just put this object into the else of the if/else clause below.

Reply:
The issue is that TS doesn't know `let error` var exists later in the code when we set the `response`.

Reply:
discussed offline - might have been mixing up languages but this is fine as is.  If we take another pass let's put this into a private function